editor/import/resource_importer_scene.cpp

```diff
@@ -2110,3 +2110,23 @@ Node *EditorSceneFormatImporterESCN::import_scene(const String &p_path, uint32_t
 Ref<Animation> EditorSceneFormatImporterESCN::import_animation(const String &p_path, uint32_t p_flags, const Map<StringName, Variant> &p_options, int p_bake_fps) {
 	ERR_FAIL_V(Ref<Animation>());
 }
+
+void ResourceImporterScene::add_importer(Ref<EditorSceneFormatImporter> p_importer) {
+	importers.reverse();
+	importers.push_back(p_importer);
+	importers.reverse();
+}
+
+void ResourceImporterScene::remove_post_importer_plugin(const Ref<EditorScenePostImportPlugin> &p_plugin) {
+	post_importer_plugins.erase(p_plugin);
+}
+
+void ResourceImporterScene::add_post_importer_plugin(const Ref<EditorScenePostImportPlugin> &p_plugin) {
+	post_importer_plugins.reverse();
+	post_importer_plugins.push_back(p_plugin);
+	post_importer_plugins.reverse();
+}
+
+void ResourceImporterScene::remove_importer(Ref<EditorSceneFormatImporter> p_importer) {
+	importers.erase(p_importer);
+}
```

Comment:
See the context: this file has definitions for `ResourceImporterScene`, then `EditorSceneFormatImporterESCN`, and you're adding more `ResourceImporterScene` definitions after that. Those definitions should be moved up.

Reply:
Moved up.